core/src/main/java/com/scalar/db/storage/common/AbstractDistributedStorage.java

```diff
@@ -0,0 +1,71 @@
+package com.scalar.db.storage.common;
```

Comment:
This PR introduces `AbstractDistributedStorage` that extracts the common logic of the `DistributedStorage` implementations.